_data/simple-icons.json

```
@@ -12741,6 +12741,16 @@
             "hex": "EA5906",
             "source": "https://webdriver.io/docs/api/"
         },
+        {
+            "title": "Webex",
+            "hex": "FFFFFF",
+            "source": "https://www.webex.com",
```

```suggestion
            "source": "https://github.com/momentum-design/momentum-ui/blob/970c5bec962a3f72e17e0b7ed69f2c38d298c405/icons-rebrand/svg/webex-helix-filled.svg",
```
@LitoMore should we change this to the link where SVG was actually fetched from?

---

Yes. We should.